app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1671,15 +1665,22 @@ table li .account_details {
 }
 
 .item-show-price {
-  border: 1px solid red;
   text-align: center;
   font-size: 52px;
   margin: 20px 0 10px 0;
   font-family: 'Cardinal', Arial, sans-serif;
 }
 
+.item-description {
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  margin-top: 10px;
+  padding: 0px 5px;
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.